lib/db/transactions.go

```diff
@@ -453,26 +456,28 @@ func (t readWriteTransaction) close() {
 	t.WriteTransaction.Release()
 }
 
-func (t readWriteTransaction) putFile(key []byte, fi protocol.FileInfo) error {
-	if fi.Blocks != nil {
-		if fi.BlocksHash == nil {
-			fi.BlocksHash = protocol.BlocksHash(fi.Blocks)
-		}
-		blocksKey := t.keyer.GenerateBlockListKey(nil, fi.BlocksHash)
-		if _, err := t.Get(blocksKey); backend.IsNotFound(err) {
+func (t readWriteTransaction) putFile(fkey []byte, fi protocol.FileInfo) error {
+	var bkey []byte
```

Comment:
Why is this declared out here? Isn't (re-) used outside of the if scope below as far as I can see.

Reply:
It was in my original which also indirected the version vector... I figured it could stay there for clarity, but it has no practical reason now, no.